### DISCUSSION TITLE
unusable only when not other support is needed

### DISCUSSION THREAD

**Discussion by techflick** [5axes/TabAntiWarping#19] [Ideas]
Would it be possible to ad an option to handle the helpers as normal body instead as support. As it is now I'm not able to use them when I needing other support. In vaguely remember that this was not the case at the beginning, maybe i'm wrong. 

Thank you for your efforts.

**Comment by 5axes** [comment]
Could be possible to add this Tab as normal geometry, but in this case would be difficult to remove the tab, 
An other issue, in this case you must activate the option : Remove mesh intersection. And in Cura even with this option the wall are modified by the addition of the Tab. 
In V5
![V5](https://user-images.githubusercontent.com/11015345/192769484-2715ed32-548a-4f92-8db8-a4d5d82d5ef6.png)
Same issue in V4
![V4](https://user-images.githubusercontent.com/11015345/192770035-52abf0b7-50f9-499b-bebb-62cdbb101bad.png)

**Reply by techflick** [reply]
harder to remove because of fusing due to not parallel lines would be a worthy tradeoff for me not able to use them at all. Besides when its only 1-2 layers it would be still reasonably easy to break or cut them away.
